prog/vm/github_runner.rb

```diff
@@ -138,13 +129,17 @@ def before_run
       github_runner.update(runner_id: response[:runner][:id], ready_at: Time.now)
       # ./env.sh sets some variables for runner to run properly
       vm.sshable.cmd("./env.sh")
-      vm.sshable.cmd("common/bin/daemonizer 'sudo -u runner /home/runner/run.sh --jitconfig #{response[:encoded_jit_config].shellescape}' runner-script")
+
+      dir = "/home/runner"
+      command = "sudo -u runner #{dir}/run.sh --jitconfig #{response[:encoded_jit_config].shellescape}"
+      command = "/bin/bash -c '#{command} > #{dir}/#{SERVICE_NAME}.stdout 2> #{dir}/#{SERVICE_NAME}.stderr'"
```

Comment:
it would be better to put logs under #{dir}/var/log instead of home directory.

Reply:
could also leave things in the systemd journal...no need to put it in files, right?

Reply:
Yeah it looks like `journalctl -u runner-script` enough for us. I removed the log redirection.